trace/CHANGELOG.md

```diff
@@ -12,3 +12,13 @@
   on `google-api-core` (#4221, #4280)
 
 PyPI: https://pypi.org/project/google-cloud-trace/0.16.0/
+
+## 0.17.0
+
+### Notable Implementation Changes
+
+- Default to use Stackdriver Trace V2 API if calling `from google.cloud import trace`.
+  Using V1 API needs to be explicitly specified in the import.(#4437)
+- Comparing `datetime.datetime`-s in trace unit tests. (#4323)
```

Comment:
@liyanhui1228 This doesn't belong in release notes (but thanks for being thorough)

Reply:
Removed.